integration/base/github/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+customer_id: "-Am-_6i5pw0u4AbspOwKN4lZUCn49u_G"
+installation_id: "RULNveQLrLj4GDum1yQhvKAcABh0GcLY"
+release_version: "0.0.4"
+set_github_contents: "replicatedhq/test-charts:/release.yml:master:.ship"
```

Comment:
this mounts in `.ship` as the github repo, so we end up with `installer/release.yml` in the expected output